### PR TITLE
OSX CMakeLists fix

### PR DESCRIPTION
**Pull request by lpenguin**
Fixed build on OSX 10.11 (El Capitan)

### CODE REVIEW

**Comment by lpenguin** [comment]
After some checks i figured out that we don't need [this section](https://github.com/KranX/Vangers/pull/99/files#diff-af3b638bc2a3e6c650974192a53c7291R53).

But `libvorbisfile` still needed.

File: CMakeLists.txt [hunk 1]
```diff
@@ -50,6 +50,15 @@ else (MINGW)
 	set(WINDOWS_RES "")
 endif (MINGW)
 
+IF (APPLE)
```
Comment (lpenguin) [inline CMakeLists.txt]: Don't need osx frameworks explicitly

**Comment by stalkerg** [comment]
Can you push again after clean?

**Comment by lpenguin** [comment]
I made new one. Sorry i didn't figured out how to edit commits in pull request

**Pull request closed without merging**